src/types/schema.d.ts

```diff
@@ -0,0 +1,162 @@
+type DateTime = string;
+type Varchar500 = string;
+type Varchar1000 = string;
+
+export interface Task {
+  task_id: number;
+  start_time: DateTime;
+  manager_params: string;
+  openwpm_version: string;
+  browser_version: string;
+}
+
+export interface Crawl {
+  crawl_id: number;
+  task_id: number;
+  browser_params: string;
+  start_time: DateTime;
+  // FOREIGN KEY(task_id) REFERENCES task(task_id)
+}
+
+export interface SiteVisit {
+  visit_id: number;
+  crawl_id: number;
+  site_url: Varchar500;
+  // FOREIGN KEY(crawl_id) REFERENCES crawl(id)
+}
+
+export interface FlashCookie {
+  id?: number;
+  crawl_id?: number;
+  visit_id?: number;
+  domain?: Varchar500;
```

Comment:
I know we use varchar in the old sqlite schema, but that's not even supported (sqlite just maps it to `text`). We should just use the generic `string` types in this schema. Really the sqlite schema should drop all of these as well.